hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/PerformanceEvaluation.java

```diff
@@ -187,11 +192,12 @@
       "Run sequential read test");
     addCommandDescriptor(SequentialWriteTest.class, "sequentialWrite",
       "Run sequential write test");
-    addCommandDescriptor(ScanTest.class, "scan",
-      "Run scan test (read every row)");
+    addCommandDescriptor(MetaWriteTest.class, "metaWrite",
+      "Run addRegion test to populate meta table; used with 1 thread");
```

Comment:
Do we want to tie this to the new cleanMeta? i.e. in the help text here say cleanMeta cleans up the writes done by this test run?